moveit_ros/hybrid_planning/local_planner/trajectory_operator_plugins/src/simple_sampler.cpp

```diff
@@ -41,21 +41,26 @@ namespace moveit::hybrid_planning
 namespace
 {
 const rclcpp::Logger LOGGER = rclcpp::get_logger("local_planner_component");
-constexpr double WAYPOINT_RADIAN_TOLERANCE = 0.2;  // rad: L1-norm sum for all joints
 }  // namespace
 
 bool SimpleSampler::initialize(const rclcpp::Node::SharedPtr& node, const moveit::core::RobotModelConstPtr& robot_model,
                                const std::string& group_name)
 {
   // Load parameter & initialize member variables
-  if (node->has_parameter("pass_through"))
+  node->declare_parameter("pass_through");
```

Comment:
This parameter isn't related to the PR, but I was touching this same area of code and noticed it wasn't working properly.